packages/turf-transform-scale/index.js

```diff
@@ -0,0 +1,69 @@
+var bbox = require('@turf/bbox');
+var meta = require('@turf/meta');
+var helpers = require('@turf/helpers');
+var centroid = require('@turf/centroid');
+var invariant = require('@turf/invariant');
+var rhumbBearing = require('@turf/rhumb-bearing');
+var rhumbDistance = require('@turf/rhumb-distance');
+var rhumbDestination = require('@turf/rhumb-destination');
+var point = helpers.point;
+var coordEach = meta.coordEach;
+var getCoords = invariant.getCoords;
+
+
+/**
+ * Moves any geojson Feature or Geometry of a specified distance along a Rhumb Line
+ * on the provided direction angle.
+ *
+ * @name scale
+ * @param {GeoJSON} geojson object to be scaled
+ * @param {number} factor of scaling, positive values greater than 0
+ * @param {boolean} [fromCenter=true] point from which the scaling will occur; it will default to the most
+ * south-west point of a `bbox` containing the geojson, if true the scaling will origin from the `centroid` of the geojson
+ * @param {boolean} [mutate=false] allows GeoJSON input to be mutated (significant performance increase if true)
+ * @returns {GeoJSON} the scaled GeoJSON object
+ * @example
+ * var poly = turf.polygon([[[0,29],[3.5,29],[2.5,32],[0,29]]]);
+ * var scaledPoly = turf.scale(poly, 3);
+ *
+ * //addToMap
+ * scaledPoly.properties = {stroke: '#F00', 'stroke-width': 4};
+ * var addToMap = [poly, scaledPoly];
+ */
+module.exports = function (geojson, factor, fromCenter, mutate) {
+    // Input validation
+    if (!geojson) throw new Error('geojson is required');
+    if (typeof factor !== 'number' && factor === 0) throw new Error('invalid factor');
+    if (!factor) throw new Error('factor is required');
+
+    // Default params
+    var isMutate = mutate === false || mutate === undefined;
```

Comment:
Is this logic correct? If `mutate` is false then `isMutate` is true? Also, if `mutate` is not defined then `isMutate` is true. The logic below says if `isMutate` is true then you clone the data. I think either the logic or the variable names are backwards. You pick.